setup.py

```diff
@@ -1,8 +1,5 @@
 #!/usr/bin/env python
 # coding=utf-8
-# :bc: Not importing unicode_literals because in Python 2 distutils,
-# some values are expected to be byte strings.
-from __future__ import absolute_import, division, print_function
```

Comment:
❕ We can also change `install_requires`:

```py
    install_requires=[
        'filters; python_version < "3.5"', # <- remove this
        'httpx',
        'phx-filters; python_version >= "3.5"', # <- change to 'phx-filters'
        'pysha3',
        'six', # <- remove this
        'typing; python_version < "3.0"',  # <- remove this
    ],
```

Also, we can probably remove this code:
```py
##
# Because of the way PyOTA declares its dependencies, it requires a
# more recent version of setuptools.
# https://www.python.org/dev/peps/pep-0508/#environment-markers
if LooseVersion(setuptools.__version__) < LooseVersion('20.5'):
    import sys
    sys.exit('Installation failed: Upgrade setuptools to version 20.5 or later')
```